scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -203,6 +216,40 @@ workflow Mutect2 {
   }
 }
 
+task GetSampleName {
+  String gatk4_jar
+  File bam
+  File bam_index
+  File? gatk4_jar_override
+
+  # Runtime parameters
+  Int? mem
+  String gatk_docker
+  Int? preemptible_attempts
+  Int? disk_space_gb
+
+  command <<<
+      # Use GATK Jar override if specified
```

Comment:
Do you really want a standalone task just to extract the sample name?

Reply:
Especially, one that will localize a bam file in the cloud...?

Reply:
> Did you do this?
I deleted "and tumor_sample_name"

Reply:
> Do you really want a standalone task just to extract the sample name?

I'm neutral.  Whatever @jsotobroad thinks is best.